src/render/texture.zig

```diff
@@ -3,6 +3,16 @@ const wlr = @import("../wlroots.zig");
 const wayland = @import("wayland");
 const wl = wayland.server.wl;
 
+pub const TextureAttribs = extern struct {
+    const GLenum = opaque {};
+    const GLuint = opaque {};
```

Comment:
why are these opaque?

Reply:
> why are these opaque?

```
typedef unsigned int GLenum;
typedef unsigned int GLuint;
```
 I wasn't sure how I'd wrap this so I just used opaque, should I declare them as u32?

Reply:
if they are 32 bit unsigned integers regardless of target architecture `u32` would be the correct type. I don't know if that's the case and don't really have time to look into it in depth.

Reply:
It seems that's the case https://www.khronos.org/opengl/wiki/OpenGL_Type

Reply:
Changed the u32.

Reply:
If that is in fact the case I would expect to see `khronos_uint32_t` in the source code not `unsigned int`

Reply:
It's still a unsigned 32bit integer right? which is equivalent to u32 so I assume the patch would work?

Reply:
`unsigned int` is not 32 bits on all targets.

Reply:
> `unsigned int` is not 32 bits on all targets.

Understood, in that case what should I annotate it with?